python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -730,29 +730,45 @@ def read_csv_header(
 class Sink(IR):
     """Sink a dataframe to a file."""
 
-    __slots__ = ("cloud_options", "kind", "options", "path")
-    _non_child = ("schema", "kind", "path", "options", "cloud_options")
+    __slots__ = ("cloud_options", "config_options", "kind", "options", "path")
+    _non_child = (
+        "schema",
+        "kind",
+        "path",
+        "config_options",
+        "options",
+        "cloud_options",
+    )
 
     kind: str
+    """The type of file to write to. Eg. Parquet, CSV, etc."""
     path: str
+    """The path to write to"""
+    config_options: ConfigOptions
+    """GPU-specific configuration options"""
+    cloud_options: dict[str, Any] | None
+    """Cloud-related authentication options, currently ignored."""
     options: dict[str, Any]
+    """Sink options from Polars"""
 
     def __init__(
         self,
         schema: Schema,
         kind: str,
         path: str,
+        config_options: ConfigOptions,
```

Comment:
General question: do people have a preference on passing the (general-purpose) `config_options` here vs. passing the more specific `config_options.parquet_options`, which I believe are the only options used here?

Reply:
I think you're doing this in https://github.com/rapidsai/cudf/pull/19186?

Reply:
Yep, that's correct. Seems like people are on board with that as a general policy, so I'll go ahead and merge that and then we can update this. IIUC, you should need to

1. Update the `Sink` class definition to reference `parquet_options` / `ParquetOptions` in slots / init / non_child
2. Change some `config_options.parquet_options.` to `parquet_options.`
3. Change how the `Sink` is constructed at https://github.com/rapidsai/cudf/pull/19015/files/efd2ff4b811a1f614130baf517ce19659b540675